lib/ansible/plugins/action/__init__.py

```diff
@@ -1372,7 +1372,7 @@ def _get_diff_data(self, destination, source, task_vars, content=None, source_fi
             elif peek_result.get('size') and C.MAX_FILE_SIZE_FOR_DIFF > 0 and peek_result['size'] > C.MAX_FILE_SIZE_FOR_DIFF:
                 diff['dst_larger'] = C.MAX_FILE_SIZE_FOR_DIFF
             else:
-                display.debug(u"Slurping the file %s" % source)
+                display.debug(u"Slurping the file %s" % destination)
```

Comment:
this is not incorrect, the slurp does act on the source file, the file to copy content from

Reply:
> ~this is not incorrect, the slurp does act on the source file, the file to copy content from~ but we are using that variable as the 'source' for slurp ... so change is correct

Did i misunderstand? Ansible use slurp module to get destination's content (see the statement after display.debug) and then use the data for comparison (compared with the source to get diff data ), so i think this debug message is for destination. i'm not clear with the word "slurping", so the above is my opinion.

Reply:
It's just a confusingly named variable, thanks for the fix!